src/osd/PrimaryLogPG.cc

```diff
@@ -4141,7 +4141,7 @@ void PrimaryLogPG::log_op_stats(const OpRequest& op,
 	   << " lat " << latency << dendl;
 
   if (m_dynamic_perf_stats.is_enabled()) {
-    m_dynamic_perf_stats.add(op, inb, outb, latency);
+    m_dynamic_perf_stats.add(op, inb, outb, now);
```

Comment:
Why the swap from ```latency``` to ```now```?

Reply:
The rationale was that `now` is more generic and allows to calculate not only `latency` but also `process_latency`. We don't need this though right now and I am not sure we will need ever. And if we need we can change this later, so I will return back to `latency`.